src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .replace(/\n/g, '')
+    .split(`;`)
+    .map(style => style.trim())
+    .filter(Boolean)
+    .map(style => style
+      .split(`:`)
+      .map(item => item.trim())
+    )
+    .reduce((acc, [key, value]) => {
+      return {
+        ...acc,
```

Comment:
I guess you could try `Object.assign` and mutate the object for better performance. But it's not critical